### PR TITLE
EVG-3678 make oauth tokens default

### PR DESCRIPTION
**Pull request by ablack12**
This is almost exactly Stu's code but I didn't see anything else that needed changing, but let me know if there's something else I should be testing or looking at.

### CODE REVIEW

**Comment by bsamek** [comment]
> We should add that and then test it in staging before it goes to prod. I can explain to you what it does when we get there

@john-m-liu I have a memory of this working in staging but not in prod. Do you remember what causes this bug to manifest?

**Comment by john-m-liu** [comment]
You need to have an expansion in the directory field of git.get_project. We can submit a patch that has that

File: service/api.go [hunk 1]
```diff
@@ -206,7 +206,13 @@ func (as *APIServer) GetProjectRef(w http.ResponseWriter, r *http.Request) {
 func (as *APIServer) GetExpansions(w http.ResponseWriter, r *http.Request) {
 	t := MustHaveTask(r)
 	h := MustHaveHost(r)
-	e, err := model.PopulateExpansions(t, h)
+	settings, err := evergreen.GetConfig()
```
Comment (tychoish) [inline service/api.go]: I think there's a setting object in the APIServer type.

File: model/task_config.go [hunk 2]
```diff
@@ -142,7 +142,12 @@ func MakeConfigFromTask(t *task.Task) (*TaskConfig, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "error finding host")
 	}
-	e, err := PopulateExpansions(t, h)
+
+	settings, err := evergreen.GetConfig()
```
Comment (tychoish) [inline model/task_config.go]: is it reasonable to pass the settings object4 into this function?

Comment (ablack12) [reply model/task_config.go]: The alternative would be to get the oauth token and pass that into PopulateExpansions instead.

**Comment by ablack12** [comment]
evergreen merge

**Pull request merged**